reference/mouse.md

```diff
@@ -0,0 +1,58 @@
+## Mouse
+
+The [Mouse](#mouse) API provides a set of functions available by default for each [Robot](robot.md) node and it is used to read the computer mouse input.
+It doesn't have any corresponding Node.
+
+> **Note** [C++, Python, Java]:
+In C++, Python and Java, the mouse functions are in a dedicated class called `Mouse`.
+In order to get the `Mouse` instance, you should call the `Robot.getMouse()` function.
+
+#### WbMouseState
+
+The state of a mouse is defined by the following structure:
+
+```c
+typedef struct {
+  // mouse buttons state
+  bool left;
+  bool middle;
+  bool right;
+  // mouse 3D position
+  double x;
+  double y;
+  double z;
+} WbMouseState;
+```
+
+The `left`, `middle` and `right` variables are matching respectively with the left, middle and right buttons of the computer mouse.
+A `true` state is matching with the pushed button, while a `false` state is matching with the released button.
```

Comment:
I'd rather say "A `true` state means the button is pressed while a `false` state means the button is released."